tests/integration/targets/inventory-v3.7/files/test-inventory-bearer-token.json

```diff
@@ -44,27 +44,28 @@
                         "custom_fields": {},
                         "description": "",
                         "device": {
-                            "display": "Test Nexus One",
-                            "id": 4,
-                            "name": "Test Nexus One"
+                            "display": "Test Nexus Child One",
+                            "id": 5,
+                            "name": "Test Nexus Child One",
+                            "url": "http://localhost:32768/api/dcim/devices/5/"
                         },
-                        "display": "Ethernet1/1",
+                        "display": "Ethernet2/1",
```

Comment:
Sorry, why were these changed? Your commit message does not indicate the reason

Reply:
The reason I ask is because it appears that you changed existing test files instead of creating new ones in the inventory that tests your specific feature.

Reply:
The inventory files were created with the .hacking.
I have just adjusted these files according to the output that was generated with it. I added virtual disks to the netbox_deploy file and have not changed any existing devices.
I used a clean install of netbox with netbox-docker and done everything according to Contribution.

Reply:
Ok thank you for the explanation